apis/r/inst/AUTHORS

```diff
@@ -0,0 +1,77 @@
+
+The 'tiledbsoma' R package has been written by a team comprised of
+members from Chan Zuckerberg Initiative (CZI) and TileDB, Inc., as
+part of the 'SOMA' ("Stack of Matrices, Annotated") initative.
+
+The repository provides detailed commit statistics across the C++,
+Python, and R components of the implementation:
+  https://github.com/single-cell-data/TileDB-SOMA/graphs/contributors
```

Comment:
This doesn't seem necessary since it's true of all GitHub repos. I think the statement above is sufficient.

Reply:
Short of enumerating authors, one can look at each repo at GitHub and page (AFAIK) it returns authors by number of commits and is hence a proxy for authors.   In any event, we like will need to iterate over this at some point anyway which you as package maintainer will likely own and then change -- but I right now I am not sure what actual change you are suggesting I make.

Can you clarify / help me out?  Then I can send one commit fixing this and above.

Reply:
It's fine to leave it in.

Reply:
Thanks. I am not totally sold on it but used it as the best I could come up with.  We could leave it as it,  it is sort of a half-way stop.  We could also filter `git log` and fetch names and emails, sort by name or height or commits or ...